README.md

```diff
@@ -335,17 +335,17 @@ lightning-cli bkpr-channelsapy <start time> <end time>
 
 ### List all outgoing satoshis currently in channels
 ```
-lightning-cli listfunds | jq '[.channels[].channel_sat] | add'
+lightning-cli listfunds | jq '[.channels[].amount_msat] | add / 1000'
```

Comment:
I think for this we want `our_amount_msat`? Otherwise this just sums up the total sat amount in opened channels instead of the outgoing liquidity.